components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -271,9 +283,19 @@ export default class ConfirmIntegration extends React.PureComponent {
                 <p>
                     <FormattedMarkdownMessage
                         id='bots.manage.created.text'
-                        defaultMessage='Your bot account **{botname}** has been created successfully. Please use the following access token to connect to the bot (see [documentation](https://mattermost.com/pl/default-bot-accounts) for further details).'
+                        defaultMessage='Your bot account **{botname}** has been created successfully. Please use the following access token to connect to the bot (see <link>documentation</link> for further details).'
```

Comment:
There's more Markdown here. Instead of adding a `strong` tag, we could just put wrap the `botname` value below like `botname: <strong>{bot.display_name || bot.username}</strong>`